### PR TITLE
EOD Day 1

### PR DESCRIPTION
**Pull request by NicholasMiron**
Tests are running on push. Routes are taking shape. Database has a shape. Queries are coming together.

### CODE REVIEW

**Review by JonathanSBecker — APPROVED** [review]
Looks good!

File: client/package.json [hunk 1]
```diff
@@ -4,14 +4,16 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "build": "webpack -d -w",
```
Comment (ryanferrer) [inline client/package.json:7]: For what reason do we not need this script to watch?

File: db/schema.sql [hunk 2]
```diff
@@ -0,0 +1,40 @@
+--use following command:
```
Comment (ryanferrer) [inline db/schema.sql:1]: Nice schema

**Review by ryanferrer — APPROVED** [review]
Just wondering what the removal of the watch flag was for. I'm guessing it's for Travis CI testing so it's not constantly running in the background.

**Review by matthewfeinberg — APPROVED** [review]

**Comment by matthewfeinberg** [comment]
Fixed launch scripts, Made client run in background, completed database schema and updated to:
not drop tables, allow only one user_id, topic_id combo in users_topics table and include username.
Added db file with test query that connects db to query, just require file in route file, added pg to package.json, Updated db for process.env variables and most recent package-locks, Installed Material-UI (now renders Material-UI components and SVG Icons), added material UI library, changed test script to run all tests (#10), main test script modified to run client and server build tests in one line such that CI should agree with this. Populated topics which are Passing tests. Completed route for seeding topics data, added axios to package.json, added .sql file to intuitively seed topics table, added comments to seedTopics route for legacy code developers, updated api call to use process.env, added search routes and db query for topics, created a db query to facilitate search, and one update to the schema (making topic_name unique) such that table cannot be seeded multiple times with the same data, updated search route to use fields instead of rows, fixed bug and added a test for search that passes.

**Pull request merged**